roles/openafs_server/README.md

```diff
@@ -18,6 +18,9 @@ description.
 
     afs_admin_principal:
     afs_admin_password: (undefined by default)
+    afs_servicekey_options:
+      - additional options passed to kadmin when creating the principal associated with the
```

Comment:
Btw, this is mis-formatted markdown. See the rendered README html (you can see it in "view file" on github).